nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -374,6 +381,222 @@ in {
         '';
       };
 
+      defaultLocale = mkOption {
+        default = null;
+        type = types.nullOr types.str;
+        example = "de_CH";
+        description = ''
+          <warning>
+           <para>This option exists since Nextcloud 21! If older versions are used,
+            this will throw an eval-error!</para>
```

Comment:
```suggestion
           <para>This option has only existed since Nextcloud 21! If an older version
            of Nextcloud is used, this will throw an error.</para>
```

Reply:
I have opted to instead remove this paragraph, as Nextcloud 22 is now the earliest version available in nixpkgs.